ext/opcache/config.m4

```diff
@@ -41,6 +41,8 @@ if test "$PHP_OPCACHE" != "no"; then
   if test "$PHP_OPCACHE_JIT" = "yes"; then
     AC_DEFINE(HAVE_JIT, 1, [Define to enable JIT])
     ZEND_JIT_SRC="jit/zend_jit.c jit/zend_jit_gdb.c jit/zend_jit_vm_helpers.c"
+    dnl LTO not compatible with global register variables
+    CFLAGS=$CFLAGS_CLEAN
```

Comment:
Indeed: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=68384

I think that the VM is also using global register variable by default when building with GCC: https://github.com/php/php-src/blob/33dd226bafa7ff4034f9e541e4456917da4173a7/Zend/zend_execute.c#L51-L74

Should we disable LTO in this case as well?

Reply:
does not seem necessary here, the build pass.